src/pages/Reviews.js

```diff
@@ -27,34 +27,33 @@ const Reviews = ({ productId }) => {
     rating: '',
     title: '',
     comment: '',
-    product: productId,
-    user,
```

Comment:
We don't need to pass `user` in the request body, because the backend is getting that value from the cookies anyawy